### PR TITLE
chore: add issue and PR templates

### PR DESCRIPTION
**Pull request by frrist**

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+### Motivation
```
Comment (lanzafame) [inline .github/pull_request_template.md:1]: Does this interfere with Github taking the long description part of a commit message and using that as the PR description?

Comment (frrist) [reply .github/pull_request_template.md:1]: I'm not sure, I suspect it does. From a quick look at PR's we've merged the past couple weeks it appears we don't use that feature very often.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+---
+name: Bug report
+about: Create a report to help us improve
+labels: kind/bug
+assignees: ''
+
+---
+### Describe the bug:
```
Comment (placer14) [inline .github/ISSUE_TEMPLATE/bug_report.md:8]:
```suggestion
### Visor Version: vx.x.x

### Describe the bug:
```

**Review by placer14 — APPROVED** [review]

File: .github/pull_request_template.md (hunk 1 above)
Comment (iand) [reply .github/pull_request_template.md:1]: I'm probably the only person who writes descriptions of changes in the commit messages. As such I'd prefer to keep the current behaviour so the commit message becomes the PR text.

Comment (frrist) [reply .github/pull_request_template.md:1]: Alright, in that case would dropping the PR template and keeping the issue template be a reasonable compromise?

Comment (iand) [reply .github/pull_request_template.md:1]: Sure

**Pull request merged**